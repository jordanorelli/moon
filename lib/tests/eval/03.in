# now we test that variables actually work

key: this is a literal value
another_key: @key

