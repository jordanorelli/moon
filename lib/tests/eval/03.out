key: this is a literal value
another_key: this is a literal value
